crates/bevy_reflect/src/type_registry.rs

```diff
@@ -87,6 +87,10 @@ impl TypeRegistry {
 
     /// Registers the type described by `registration`.
     pub fn add_registration(&mut self, registration: TypeRegistration) {
+        if self.registrations.contains_key(&registration.type_id()) {
```

Comment:
I think this should probably more intelligently merge the type data of the registration without overwriting existing data/registrations rather than skipping, but for now I'm fine with this.

Reply:
Agreed here, but that can come later.